docs/includes/generated_docs/cli.md

```diff
@@ -0,0 +1,593 @@
+```
+ehrql [--help] [--version] COMMAND_NAME ...
+```
+The command line interface for ehrQL, a query language for electronic health
+record (EHR) data.
+
+<div class="attr-heading" id="ehrql.command_name">
+  <tt>COMMAND_NAME</tt>
+  <a class="headerlink" href="#ehrql.command_name" title="Permanent link">🔗</a>
+</div>
+<div markdown="block" class="indent">
+Name of the sub-command to execute.
+<div class="attr-heading">
+  <a href="#generate-dataset"><tt>generate-dataset</tt></a>
+</div>
+<p class="indent">
+Take a dataset definition file and output a dataset.
+</p>
+
+<div class="attr-heading">
+  <a href="#generate-measures"><tt>generate-measures</tt></a>
+</div>
+<p class="indent">
+Take a measures definition file and output measures.
+</p>
+
+<div class="attr-heading">
+  <a href="#sandbox"><tt>sandbox</tt></a>
+</div>
+<p class="indent">
+Start the ehrQL sandbox environment.
+</p>
+
+<div class="attr-heading">
+  <a href="#dump-example-data"><tt>dump-example-data</tt></a>
+</div>
+<p class="indent">
+Dump example data for the ehrQL tutorial to the current directory.
+</p>
+
+<div class="attr-heading">
+  <a href="#dump-dataset-sql"><tt>dump-dataset-sql</tt></a>
+</div>
+<p class="indent">
+Output the SQL that would be executed to fetch the results of the dataset
+definition.
+</p>
+
+<div class="attr-heading">
+  <a href="#create-dummy-tables"><tt>create-dummy-tables</tt></a>
+</div>
+<p class="indent">
+Generate dummy data for a dataset and write out tables as CSV.
+</p>
+
+<div class="attr-heading">
+  <a href="#assure"><tt>assure</tt></a>
+</div>
+<p class="indent">
+Experimental command for running assurance tests.
+</p>
+
+<div class="attr-heading">
+  <a href="#test-connection"><tt>test-connection</tt></a>
+</div>
+<p class="indent">
+Internal command for testing the database connection configuration.
+</p>
+
+</div>
+
+<div class="attr-heading" id="ehrql.help">
+  <tt>-h, --help</tt>
+  <a class="headerlink" href="#ehrql.help" title="Permanent link">🔗</a>
+</div>
+<div markdown="block" class="indent">
+show this help message and exit
+
```

Comment:
nit: Again, not a big problem, but I think there's an extra empty line being added before the closing `div`.

Reply:
It turns out that this is because there's a slot in the argument template which is only used for subcommands, and when that's empty we get an empty line. Given that the Markdown compiles to exactly the same HTML either way I'm inclined not to worry about it.

Reply:
Yep, also all fine, it was more a "in case you hadn't seen". But it's not a big problem.